README.md

```diff
@@ -38,15 +38,15 @@ Gardener takes part in the [Certified Kubernetes Conformance Program](https://ww
 
 Continuous conformance test results of the latest stable Gardener release are uploaded regularly to the CNCF test grid:
 
-| Provider/K8s | v1.25 | v1.24 | v1.23 | v1.22 | v1.21 | v1.20 | v1.19 | v1.18 | v1.17 |
```

Comment:
As we discussed with you on the phone https://github.com/gardener/gardener/blob/4823157cccf0782e02a9ffbeb4a79fde2a85849c/pkg/operation/botanist/component/kubeapiserver/secrets.go#L188-L199 has to be adapted in the way we agreed and https://github.com/gardener/gardener/blob/e4e5c9f947c2e941fb9a6b93adf387f573e405af/pkg/operation/botanist/component/kubeapiserver/kube_apiserver.go#L134-L135 has to be cleaned up from the kubeapiserver component.

Reply:
It should be now addressed with the latest force push. Thanks!